pkgs/desktops/gnome-3/extensions/gtile/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-gtile";
+  version = "34";
+
+  src = fetchFromGitHub {
+    owner = "gTile";
+    repo = "gTile";
+    rev = "V${version}";
+    sha256 = "1yka4acmrykh4vdd89yvn6yv89c6q16r72lgi3j48my3jghwjiwp";
+  };
+
+  uuid = "gTile@vibou";
+
+  nativeBuildInputs = [ glib ];
+
+  installPhase = ''
+    mkdir -p $out/share/gnome-shell/extensions/$uuid
+    cp -r * $out/share/gnome-shell/extensions/$uuid
+
+    schemadir=${glib.makeSchemaPath "$out" "${pname}-${version}"}
```

Comment:
Do the schemas installed to the extension directory as done on the first line not work?

Reply:
They work in the way that i can see extension settings from Extensions app but cannot see them if I run dconf-editor. If i copy schemas to schemadir and add extension package to services.xserver.desktopManager.gnome3.sessionPath list, then it works (that schemadir is appended in variable XDG_DATA_DIR).

Reply:
Yeah, I do not think the schemas of extensions are expected to be seen in dconf editor. And I am still not sure if dconf-editor is even compatible with Nix (since it relies on global data like `sessionPath`).

Reply:
From description of `sessionPath` option:
> Additional list of packages to be added to the session search path. Useful for GNOME Shell extensions or GSettings-conditional autostart. Note that this should be a last resort; patching the package is preferred (see GPaste).

I've took a look at gpaste and been inspirited with usage of `glib.makeSchemaPath`.

Reply:
My motivation was to get option names using dconf-editor, without looking at the xml schema of extension.